pkg/kubelet/config/file.go

```diff
@@ -189,8 +189,7 @@ func (s *sourceFile) extractFromFile(filename string) (pod *v1.Pod, err error) {
 		return pod, nil
 	}
 
-	return pod, fmt.Errorf("%v: read '%v', but couldn't parse as pod(%v).\n",
-		filename, string(data), podErr)
+	return pod, fmt.Errorf("%v: couldn't parse as pod(%v), please check config file.\n", filename, podErr)
```

Comment:
In other places of file.go, the log uses `config file`, so I' prefer to use `config file` here also. Michael, WDYT?

L145: `glog.V(1).Infof("Can't process config file %q: %v", path, err)`
L157: `glog.V(3).Infof("Reading config file %q", filename)`